index.js

```diff
@@ -99,8 +98,10 @@ app.post('/', function(req, response) {
   			// answer fetched from susi
 			ans = (JSON.parse(body1)).answers[0].actions[0].expression;
   			
-			// setting options to request the chat api of viber.
-			var options =   {
+			// checking type of json response
+			if((JSON.parse(body1)).answers[0].actions.length == 3 &&  (JSON.parse(body1)).answers[0].actions[2].type == "map" ){
```

Comment:
because json response that we get for location has three actions while for message queries here is one action. I have to check type from third action for location and if I won't check it and there is not 3rd action for query then it will give error.